plugins/resteasy-http/src/main/java/org/radargun/http/service/AbstractRESTEasyService.java

```diff
@@ -61,6 +63,12 @@
    @Property(doc = "Http headers for request. Default is null", complexConverter = KeyValueProperty.KeyValuePairListConverter.class)
    private List<KeyValueProperty> httpHeaders;
 
+   @Property(doc = "If the REST client will ignore the https certificates")
```

Comment:
Ignoring https certificates and using insecure http protocol are two different things. From my understanding insecure property does the latter while for ignoring https certificates there is `trustAll` bellow.

Reply:
Maybe change to:
* protocol=(http|https)
* trust-all=(true|false)

WDYT?

Reply:
Sounds good